t/t1001-mf-priority-basic.t

```diff
@@ -90,7 +90,8 @@ test_expect_success 'submit a job with default urgency' '
 	cat <<-EOF >job1.expected &&
 	45321
 	EOF
-	test_cmp job1.expected job1.test
+	test_cmp job1.expected job1.test &&
```

Comment:
Suggestion: add a commit body to this commit that explains the problem the commit is trying to solve (i.e. why the missing `flux job cancel` is problematic), then how that problem is solved by the commit.